src/Microsoft.AspNet.Mvc.Localization/HtmlLocalizerOfT.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using Microsoft.Framework.Localization;
+
+namespace Microsoft.AspNet.Mvc.Localization
+{
+    /// <summary>
+    /// Represents an <see cref="IHtmlLocalizer"/> that provides strings for <see cref="TResourceSource"/>.
+    /// </summary>
+    /// <typeparam name="TResourceSource">The <see cref="System.Type"/> to provide strings for.</typeparam>
+    public class HtmlLocalizer<TResourceSource> : IHtmlLocalizer<TResourceSource>
```

Comment:
Why not inherit from `HtmlLocalizer` and reuse most, if not all of that implementation?